src/lib/validation.ts

```diff
@@ -0,0 +1,54 @@
+import { btcChainId } from '@shapeshiftoss/caip'
+import { PublicKey } from '@solana/web3.js'
+import { solanaChainId } from 'constants/caip'
+import debounce from 'lodash/debounce'
+import WAValidator from 'multicoin-address-validator'
+import { isAddress } from 'viem'
+
+import { isEvmChainId } from './utils'
+
+const validateAddressSync = (address: string, chainId: string): boolean => {
```

Comment:
Since this just returns a `boolean`, maybe

```suggestion
const isValidAddressSync = (address: string, chainId: string): boolean => {
```

Reply:
Absolutely! 40fd051